src/Twig.php

```diff
@@ -137,11 +142,11 @@ public function render(ResponseInterface $response, $template, $data = [])
      * Create a loader with the given path
      *
      * @param array $paths
-     * @return \Twig_Loader_Filesystem
+     * @return \Twig\Loader\FilesystemLoader
      */
     public function createFilesystemLoader(array $paths)
     {
-        $loader = new \Twig_Loader_Filesystem();
+        $loader = new \Twig\Loader\FilesystemLoader();
```

Comment:
Available in Twig 1.0+. See https://github.com/twigphp/Twig/blob/1.x/lib/Twig/Loader/Filesystem.php#L2C1-L3C34